### PR TITLE
Added migrate size and code flag.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/MigrateBatch.java [hunk 1]
```diff
@@ -1,23 +0,0 @@
-package org.opentestsystem.rdw.ingest.migrate.reporting;
```
Comment (agorina) [inline migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/MigrateBatch.java]: This seems to be redundant now. Migrate has all that is needed.

File: migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/MigrateJobParameters.java [hunk 2]
```diff
@@ -16,14 +16,16 @@
     static final String FirstAt = "firstAt";
     static final String LastAt = "lastAt";
     static final String MigrateCodes = "migrateCodes";
+    static final String BatchSize = "size";
 
-    public MigrateJobParameters(final Instant firstAt, final Instant lastAt, final boolean migrateCodes) {
+    public MigrateJobParameters(final Instant firstAt, final Instant lastAt, final boolean migrateCodes, final int batchSize) {
         super(new JobParametersBuilder()
```
Comment (agorina) [inline migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/MigrateJobParameters.java]: I know, 4 parameters = builder, but it seems like an overkill in this case.

File: migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/MigrateReportingConfiguration.java [hunk 3]
```diff
@@ -74,7 +74,7 @@ public JobExecutionListener listener() {
      * </ol>
      */
     @Bean
-    public Job migrateReportingJob(final Step truncateStageStep,
```
Comment (agorina) [inline migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/MigrateReportingConfiguration.java]: Unrelated - spring batch wants each step to have a unique name. It was giving a warning about it.

File: migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/ExecutionParams.java [hunk 4]
```diff
@@ -6,7 +6,7 @@
  */
 public final class ExecutionParams {
 
-    public static final String batch = "batch";
+    public static final String migrate = "migrate";
```
Comment (malaffoon) [inline migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/ExecutionParams.java]: this poor class exists for one lousy string and it keeps changing

File: migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/MigrateJobParameters.java (hunk 2 above)
Comment (malaffoon) [inline migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/MigrateJobParameters.java]: fine by me

File: migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/ExecutionParams.java (hunk 4 above)
Comment (agorina) [inline migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/ExecutionParams.java]: I know, I had the same thought. 
But this 'one lousy string' needs a home too :)

**Pull request merged**